rai/client.go

```diff
@@ -1713,3 +1713,65 @@ func (c *Client) ListSnowflakeDatabaseLinks(
 	}
 	return result, nil
 }
+
+//
+// Snowflake Data Steams
+//
+
+func (c *Client) CreateSnowflakeDataSteam(
+	integration, database, schema, objectName, role, warehouse string, isView bool,
+	raiDatabase, relation string,
+	columns []SnowflakeSchemaColumn, creds *SnowflakeCredentials,
+) (*SnowflakeDataStream, error) {
+	var result SnowflakeDataStream
+	dbLink := fmt.Sprintf("%s.%s", database, schema)
+	path := makePath(PathIntegrations, integration, "database-links", dbLink, "data-streams")
+	req := createSnowflakeDataStreamRequest{}
```

Comment:
perhaps better to get the caller to pass this object? instead of having 8 separate args here